cmd/internal/find/primary/numeric/predicate.go

```diff
@@ -25,6 +56,7 @@ func ParsePredicate(str string, parsers ...Parser) (Predicate, int, error) {
 		panic("numeric.ParsePredicate called without any parsers")
 	}
 
+	// TODO: Introduce "+="/"-=" to represent ">="/"<="?
```

Comment:
We're using +/- primarily to avoid shell piping operators? I guess the symmetry makes sense then.

Reply:
I used +/- b/c BSD/GNU find uses them, but I'm guessing their reasoning for using them is as you said -- to avoid the shell piping operators.

I can add the symmetry as a follow-up.